modules/apps/object/object-rest-impl/rest-openapi.yaml

```diff
@@ -28,6 +28,12 @@ components:
                     type: object
         FileEntry:
             properties:
+                fileBase64:
+                    # @review
+                    description:
+                        optional field with the content of the document in Base64, can be embedded with nestedFields
```

Comment:
@ccorreagg how will users know what to set as nested field? I would not have a clue. Is there a better way to document it?

Reply:
I guess by including the name of the nested field that must be used?. The description added is the one that is always included in the OpenAPI file. Maybe this is a special case and would need extra description. Or we could even think about updating dinamically the `nestedFields` description with the possible nested fields?

Reply:
Well, I updated the PR with a new commit